GVRf/Framework/jni/shaders/material/mesh_animation_shader.cpp

```diff
@@ -0,0 +1,133 @@
+#include "mesh_animation_shader.h"
```

Comment:
maybe @danke-sra  forgot to remove it . That shader is used at @fredduarte's implementation